changes for adding additional information for invalid status in log

rasamala83 opened the pull request:
The earlier condition was RAC data tm(time value in millis) change or status change we allow proceed. Now, the code has changed with the "and" condition because if there is no change in status and TM then we can skip RAC_MAINT_CHANGE event.

shtiencode reviewed (commented):
I think we need the "or" operator because it is valid scenario that status changes but status time remains. It may be necessary to have respective checks (if condition) for time and status.

rasamala83 commented:
> I think we need the "or" operator because it is valid scenario that status changes but status time remains. It may be necessary to have respective checks (if condition) for time and status.

Reverted the condition and added status check for the latest row data. This stop trigger RACMAINT_INFO_CHANGE for status 'U'.

ModEtchFill commented:
The Java CI failure could be from tests/devdocker

[5/6] RUN go get github.com/go-sql-driver/mysql
18:53:45.248 [main] DEBUG com.paypal.hera.jdbc.UtilHeraBox - #10 1.282 # filippo.io/edwards25519
Error: 18:53:45.248 [main] DEBUG com.paypal.hera.jdbc.UtilHeraBox - #10 1.282 src/filippo.io/edwards25519/scalar.go:166:53: cannot convert x (type []byte) to type *[32]byte

I've made https://github.com/paypal/hera/pull/381 to try to fix it
